include/mbedtls/ssl_ciphersuites.h

```diff
@@ -364,6 +364,19 @@ typedef enum {
 #define MBEDTLS_KEY_EXCHANGE_SOME_ECDHE_ENABLED
 #endif
 
+/* TLS 1.2 key exchanges using ECDH(E)*/
+#if defined(MBEDTLS_KEY_EXCHANGE_SOME_ECDH_ENABLED) || \
+    defined(MBEDTLS_KEY_EXCHANGE_SOME_ECDHE_ENABLED)
+#define MBEDTLS_KEY_EXCHANGE_SOME_ECDH_OR_ECDHE_1_2_ENABLED
+#endif
+
+/* TLS 1.2 or 1.3 key exchanges using ECDH(E) */
+#if defined(MBEDTLS_KEY_EXCHANGE_SOME_ECDH_OR_ECDHE_1_2_ENABLED) || \
+    defined(MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_EPHEMERAL_ENABLED) || \
+    defined(MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_PSK_EPHEMERAL_ENABLED)
```

Comment:
Mmm that's strange: even thought the code I have locally seems to be aligned with the one on the server, my definition is different:
```
/* TLS 1.2 or 1.3 key exchanges using ECDH or ECDHE */
#if defined(MBEDTLS_KEY_EXCHANGE_SOME_ECDH_OR_ECDHE_1_2_ENABLED) || \
    defined(MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_SOME_ECDHE_ENABLED)
#define MBEDTLS_KEY_EXCHANGE_SOME_ECDH_OR_ECDHE_ANY_ENABLED
#endif
```

But a part from this, since the definition is for `SOME_ECDH_OR_ECDHE_ANY` it seems to me that `SOME_EPHEMERAL` might be too generic. I think it might be better to stick with `ECDHE`. Wdyt?

Reply:
Right, we should stick with ECDHE. I was merely observing that the last two lines in the definition I was looking to could be shortened to one line, but was missing the point that it wasn't the best anyway.

Also, it looks like I was commenting on an old commit and the definition has been updated in the meantime.